stages/org.osbuild.firewall

```diff
@@ -92,6 +116,15 @@ def main(tree, options):
                        list(map(lambda x: f"--remove-service={x}", disabled_services)),
                        check=True)
 
+    for zone_sources in sources:
+        zone = zone_sources["zone"]
```

Comment:
`zone` is not marked as `required` in line 72, but is unconditionally used.

Reply:
yeah, the thing here is just you can do:

$firewalld-offline-cmd --zone="" --add-source 192.157.134.44 

and the above applies to the default zone anyway

Reply:
ok, I understand now! But then it should be `zone = zone_sources.get("zone", "")`, no?

Reply:
fixed!